docs/source/frameworks/transformers.rst

```diff
@@ -87,6 +87,29 @@ such as tokenizers, preprocessors, and feature extractors, can also be saved as
 
 To load the pre-trained instances for testing and debugging, use :code:`bentoml.transformers.load_model` with the same tags.
 
+Starting from BentoML version 1.1.9, importing pre-trained Transformer models from Hugging Face has been further streamlined.
+You can choose to use the new ``bentoml.transformers.import_model`` function to import models directly into the BentoML Model Store without the overhead of loading them into memory.
```

Comment:
I think in some edge cases we still need to load it into memory? @MingLiangDai

Reply:
Yup, so far I think the only identified issue is `trust_remote_code=True`. Mentioned in another comment.